src/config.c

```diff
@@ -253,6 +253,7 @@ config_property_definition _twitchDefinitions[] = {
 config_property_definition _networkDefinitions[] = {
 	{ offsetof(network_configuration, player_name),						"player_name",					CONFIG_VALUE_TYPE_STRING,		{.value_string = "Player" },	NULL					},
 	{ offsetof(network_configuration, default_port),					"default_port",					CONFIG_VALUE_TYPE_UINT32,		NETWORK_DEFAULT_PORT,			NULL					},
+	{ offsetof(network_configuration, default_password),				"default_password",			    CONFIG_VALUE_TYPE_STRING,       {.value_string = NULL     },   NULL                    },
```

Comment:
You have mixed spaces and tabs here, please make sure you line them up correctly.